packages/plugins/graphql-plugin-auth/README.md

```diff
@@ -26,6 +26,19 @@ const neoSchema = new Neo4jGraphQL({
         }),
     },
 });
+
```

Comment:
Thanks for adding this info to the README.md.
Let's also add this to the Documentation in the setup page for Auth: https://github.com/neo4j/graphql/blob/dev/docs/modules/ROOT/pages/auth/setup.adoc

Reply:
resolved by 7ccc4ae3555a648b572aa4db8f5e04d58f5ec51b